### ISSUE TITLE
incorrect compilation in gulp-csso

### ISSUE THREAD

**Issue by ademaro** [css/csso#230]
gulp-csso (last version 1.0.0) compiles these lines wrong:
https://github.com/twbs/bootstrap/blob/master/less/forms.less#L54-L55
```css
input[type="radio"],
input[type="checkbox"] {
  margin: 4px 0 0;
  margin-top: 1px \9; // IE8-9
  ...
}
```
into this:
```css
input[type="radio"],input[type="checkbox"]{margin:1px 0 0}
```
instead of:
```css
input[type="radio"],input[type="checkbox"]{margin:4px 0 0;margin-top:1px \9}
```

**Comment by rinatio** [comment]
+1

**Comment by lahmatiy** [comment]
Fixed by #166

**Comment by ademaro** [comment]
Many thanks!

**Comment by lahmatiy** [comment]
Actually it isn't fixed completely :-/ There is another issue #231
